sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -371,6 +371,10 @@ public static class StatusCodes {
 
         public static final int SERVICE_UNAVAILABLE = 503;
         public static final int INTERNAL_SERVER_ERROR = 500;
+        /**
+         * Used by {@link RequestCancelledException} to indicate a client side request cancellation
+         */
+        public static final int REQUEST_CANCELLED = 499;
     }
```

Comment:
No 499 - like discussed in end2end timeout PR

Reply:
Oops this is stale code as I branched this off of my earlier PR. Will remove